package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "quizzer",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "NODE_ENV=production node dist/",
+    "dev": "cross-env NODE_ENV=development ts-node-dev server/index.ts",
+    "test:routes": "cross-env NODE_ENV=test jest ./__test__/backend/routes.test.ts",
+    "test:db": "cross-env NODE_ENV=test jest ./__test__/backend/db.test.ts",
+    "test:frontend": "cross-env NODE_ENV=test jest ./__test__/frontend",
+    "build": "tsc -p tsconfig.json",
+    "prepare": "husky install",
+    "client": "cd client/ && npm start",
+    "heroku-postbuild": "cd client/ && npm i && npm run build"
```

Shouldn't this be like that?

```suggestion
    "heroku-postbuild": "npm run build && cd client/ && npm i && npm run build"
```